src/visitor/FoldConstantExprVisitor.cpp

```diff
@@ -115,6 +115,11 @@ void FoldConstantExprVisitor::visit(FunctionCallExpression *expr) {
     canBeFolded_ = canBeFolded;
 }
 
+void FoldConstantExprVisitor::visit(AggregateExpression *expr) {
+    // TODO : impl AggExpr foldConstantExprVisitor
```

Comment:
We could do folding for the args expr of agg.

Reply:
Already add.